### PR TITLE
move packages for v2

### PR DESCRIPTION
**Pull request by markelliot**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ repositories {
 }
 
 allprojects {
-    group 'com.palantir.hadoop-crypto'
+    group 'com.palantir.hadoop-crypto2'
```
Comment (markelliot) [inline build.gradle:29]: probably end up having to mess with some stuff on bintray out of this

**Comment by markelliot** [comment]
not sure if we actually have to move all the crypto stuff to a `.crypto2` or just the hadoop stuff

**Comment by uschi2000** [comment]
I'd probably just move it all.

**Comment by markelliot** [comment]
This PR moves everything but seek-io, which I intend to move to another project entirely (PR is in to our OSS management repo to get that provisioned.

**Review by ellisjoe — APPROVED** [review]

**Pull request merged**